infrastructure/index.ts

```diff
@@ -213,6 +213,7 @@ const baseCacheBehavior = {
 // https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#limits_cloudfront
 const distributionArgs: aws.cloudfront.DistributionArgs = {
     enabled: true,
+    staging: false, // Attempt to work around https://github.com/pulumi/pulumi-aws/issues/5229
```

Comment:
Could you elaborate a little on what's not currently working and why this fixes it?

Reply:
We are on AWS v5, and would like to be on v6 (latest and greatest, etc.). Just upgrading (attempted in https://github.com/pulumi/registry/pull/6730) tries to replace the distribution due to https://github.com/pulumi/pulumi-aws/issues/5229. Setting `staging: false` proactively is the suggested work-around.

Reply:
Ah gotya - so this property is added, and is required in v6, so we'll add it now and it shouldn't change anything in v5 .. but lets us also move to v6.

Reply:
Hopefully.